src/applications/disability-benefits/all-claims/containers/FormSavedPage.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import FormSaved from '../../../../platform/forms/save-in-progress/FormSaved';
+
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+
+const FormSavedPage = props => {
+  const itfExpirationDate = moment(props.itf.currentITF.expirationDate).format(
```

Comment:
it's not possible for someone to get the form into a state where it's saved without them also having an active ITF, right?

Reply:
Technically yes? They'd have to be active on the form so their session doesn't expire by the time they reach their ITF expiration. In that event, though, it'd be "fine" as far as the data's concerned--that is, we won't be calling `currentITF of undefined`.